blkio_test.go

```diff
@@ -24,32 +24,38 @@ import (
 	v1 "github.com/containerd/cgroups/stats/v1"
 )
 
-const data = `   7       0 loop0 0 0 0 0 0 0 0 0 0 0 0
-   7       1 loop1 0 0 0 0 0 0 0 0 0 0 0
-   7       2 loop2 0 0 0 0 0 0 0 0 0 0 0
-   7       3 loop3 0 0 0 0 0 0 0 0 0 0 0
-   7       4 loop4 0 0 0 0 0 0 0 0 0 0 0
-   7       5 loop5 0 0 0 0 0 0 0 0 0 0 0
-   7       6 loop6 0 0 0 0 0 0 0 0 0 0 0
-   7       7 loop7 0 0 0 0 0 0 0 0 0 0 0
-   8       0 sda 1892042 187697 63489222 1246284 1389086 2887005 134903104 11390608 1 1068060 12692228
-   8       1 sda1 1762875 37086 61241570 1200512 1270037 2444415 131214808 11152764 1 882624 12409308
-   8       2 sda2 2 0 4 0 0 0 0 0 0 0 0
-   8       5 sda5 129102 150611 2244440 45716 18447 442590 3688296 67268 0 62584 112984`
+const data = `major minor  #blocks  name
+
```

Comment:
Is this line guaranteed to be an empty? Is there a kernel documentation about this?

Reply:
So, the kernel documentation on the subject [is rather scarce](https://man7.org/linux/man-pages/man5/proc.5.html).  In the code 2 lines are guaranteed and were there for quite a while (at least from [2.6.32](https://elixir.bootlin.com/linux/v2.6.32/source/block/genhd.c#L733)).  

PS. That said, [/proc/diskstats](https://elixir.bootlin.com/linux/v5.12.1/source/block/genhd.c#L1256) has eventually disabled this header.  I'm not sure we want to complicate the code just in case they ever do that for /proc/partitions.